.githooks/pre-commit

```diff
@@ -40,4 +40,3 @@ do
         "$format" -i -style=file "$file"
     fi
 done
-
```

Comment:
Irrelevant parts appear to have been modified.

Reply:
removed as per your comment